package.json

```diff
@@ -9,7 +9,13 @@
     "export": "next build && next export",
     "format": "prettier --write \"./components/**\" \"./types/**\" \"./pages/**\" \"./context/**\" --ignore-path ./.gitignore",
     "lint": "eslint \"./components/**\" \"./pages/**\" \"./context/**\" \"./types/**\"",
-    "lint:fix": "yarn lint --fix"
+    "lint:fix": "yarn lint --fix",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.js, *.ts, *.jsx, *.tsx": [
+      "npm run lint"
+    ]
```

Comment:
If we use `lint:fix`, are there any issues in that?

Reply:
It won't have any effect since the hook is going to be run before committing.